stl/inc/xutility

```diff
@@ -4701,7 +4710,7 @@ struct _Trivial_cat {
         _Same_size_and_compatible && is_trivially_constructible_v<_Dest, _SourceRef>;
 
     static constexpr bool _Bitcopy_assignable =
-        _Same_size_and_compatible && is_trivially_assignable_v<_DestRef, _SourceRef>;
+        _Same_size_and_compatible && _Is_trivially_assignable_returning_same_reference_v<_DestRef, _SourceRef>;

```

Comment:
Look like that I should un-recover scalar types.

Reply:
Nope. But I think I was fooled. I don't think we should need to suppress these warnings for unevaluated operands. Reported DevCom-10906899.